sigmf-spec.md

```diff
@@ -231,7 +230,7 @@ the `capture` object:
 |`sample_start`|true|uint|The sample index at which this segment takes effect.|
 |`frequency`|false|double|The center frequency of the signal in Hz.|
 |`sample_rate`|false|double|The sample rate of the signal in Samples per Second.|
-|`time`|false|string|An ISO-8601 formatted string indicating the timestamp of the sample index specified by `sample_start`.|
+|`datetime`|false|string|An ISO-8601 formatted string indicating the timestamp of the sample index specified by `sample_start`, using the UTC/Zulu timezone.|
```

Comment:
I'd like this even better if it said exactly which of the string formats ISO 8601 defines is allowed here, rather than the non-specific "ISO-8601 formatted string".

I don't know what the proper wording is and ISO 8601 is not freely available so I can't be more specific. Wikipedia used the phrase “combined date and time”. One could write a template like “`YYYY-MM-...`” and rely on ISO 8601 to disambiguate the details.